runtime/lua/vim/lsp/util.lua

```diff
@@ -839,7 +839,7 @@ function M.convert_signature_help_to_markdown_lines(signature_help, ft, triggers
   local active_hl
   local active_signature = signature_help.activeSignature or 0
   -- If the activeSignature is not inside the valid range, then clip it.
```

Comment:
Just here, stating this is for backwards compatibility with 3.15

Reply:
Done